qemu/tests/cfg/tpm_verify_device.cfg

```diff
@@ -5,12 +5,17 @@
     kill_vm = yes
     tpms = tpm0
     tpm_version_tpm0 = 2.0
+    tpm_model_tpm0 = tpm-crb
     x86_64:
         only q35
         only ovmf
         required_qemu= [4.2.0,)
     ppc64le, ppc64:
         required_qemu= [5.0.0,)
+        tpm_model_tpm0 = tpm-spapr
```

Comment:
You mean that the `tpm_model = tpm-spapr` also works for passthrough test scenarios? and the ppc and aarch64 has supported passthrough device?
If not, I suggest splitting the `tpm_model` into emulator and passthrough in order to show clear.
Like:
Line 16 ~ 18:
```
    aarch64:
        required_qemu = [5.1.0,)
```

Add the below behind line 35:
```
aarch64:       
        tpm_model_tpm0 = tpm-tis-device
```

Reply:
`tpm-tis-device` and `tpm-spapr` are the only TPM device of aarch64 and ppc64le, ppc64le supports vTPM passthrough but aarch64 not yet, so it is better to assign a default device for each architecture rather than specify tpm-crb.
If restore them to the original place, `with_passthrough` will use the wrong device on non-x86.